jet_images_ml.py

```diff
@@ -259,9 +259,9 @@ def optimize(num_iterations, starting_iteration=0):
 
             summary_value_val = session.run(merged_summary, feed_dict_validate)
             eval_writer.add_summary(summary_value_val, i)
-	    #classifier_features=session.run([flat_fc1],feed_dict=feed_dict_train)
-	    #classifier.append(classifier_features)
-	    labels.append(y_true_batch)
+	    classifier_features=session.run([activated_fc1],feed_dict=feed_dict_train)
```

Comment:
I have a doubt here! We are considering only the last batch in every epoch and extracting the features. IS it the right way to do?